### ISSUE TITLE
Genome-wide scan or only SNPs of interests

### ISSUE THREAD

**Issue by Yiguan** [35ajstern/clues#11]
Hi Aaron,

Thanks for providing such an amazing tool. I basically have two questions when applying `clues` to my real genome data.

- which SNPs in under selection?

- what's the selection coefficicents for the SNPs unders selection?

To answer these questions, does that mean I need to scan each SNP along the whole genome? Then for each SNP, I could have the scan results, e.g:

```
Loading data and initializing model...
(100, 1) (100, 21)
Optimizing likelihood surface using Nelder-Mead...
        (Importance sampling with M = 100 Relate samples)

##########

logLR: 0.2364

MLE:
========
epoch   selection
0-1000  0.00254
0
```

And then  `logLR: 0.2364` indicates the likelihood of the SNP under selection? and `0.00254` being the selection coefficient?

If so, then I think the biggest challenge is compuation if all the SNPs are scanned individually, which might take many weeks...

In short for my question, is 'clues' suitable for genome-wide scan without any knowledge of candidate SNPs?

Thanks very much!

Yiguan
